### PR TITLE
u3d/licenses: add feature to display licenses

### PR DESCRIPTION
**Pull request by lacostej**
### Pull Request Checklist

- [x] My pull request has been rebased on master
- [x] I ran `bundle exec rspec` to make sure that my PR didn't break any test
- [x] I ran `bundle exec rubocop` to make sure that my PR is inline with our code style
- [x] I have read the [code of conduct](https://github.com/DragonBox/u3d/blob/master/CODE_OF_CONDUCT.md)

### Pull Request Description

u3d/licenses: add feature to display licenses

### CODE REVIEW

**Comment by lacostej** [comment]
To test on Windows / Linux

File: lib/u3d/commands_generator.rb [hunk 1]
```diff
@@ -189,6 +189,14 @@ def run
         end
       end
 
+      command :licenses do |c|
+        c.syntax = "u3d licenses>"
```
Comment (niezbop) [inline lib/u3d/commands_generator.rb]: Typo

File: lib/u3d/unity_license.rb [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+    def initialize(path: nil, fields: nil)
+      @path = path
+      @fields = fields || {}
+    end
+
+    def [](index)
+      @fields[index]
+    end
+
+    # petit cachotier va!
```
Comment (niezbop) [inline lib/u3d/unity_license.rb]: 👍

File: lib/u3d/commands_generator.rb [hunk 3]
```diff
@@ -189,6 +189,14 @@ def run
...
+        c.description = 'Prints Unity license information.'
+        c.action do |args, _options|
+          Commands.licenses(args: args)
```
Comment (niezbop) [inline lib/u3d/commands_generator.rb]: Idem, you could omit args if you don't need them further down

File: lib/u3d/commands.rb [hunk 4]
```diff
@@ -195,6 +195,12 @@ def credentials(args: [])
         end
       end
 
+      def licenses(_args)
```
Comment (niezbop) [inline lib/u3d/commands.rb]: You could omit the args if you don't need them.

File: lib/u3d/commands_generator.rb (hunk 3 above)
Comment (lacostej) [reply lib/u3d/commands_generator.rb]: I preferred to keep them to be similar to the other commands. The idea was to maybe add feature like add license later.

Comment (niezbop) [reply lib/u3d/commands_generator.rb]: Well not all of our commands have `args` as a parameter (list, available and dependencies don't for instance) and dependencies does not have `options` either. But it's really minor nitpicking, so do as you feel best

**Pull request merged**